HISTORY.md

```diff
@@ -14,8 +16,18 @@
 
 ## Bug fixes
 
+* Fixed bug in setting a fill value for var-sized attributes.
+
 ## API additions
 
+### C API
+
+* Added functions `tiledb_attribute_{set,get}_fill_value` to get/set default fill values
```

Comment:
I believe we should call this `default_value`, it is the commonly used database term.

Reply:
HDF5 and Zarr call it "fill value":
* https://support.hdfgroup.org/HDF5/doc_resource/H5Fill_Values.html#:~:text=A%20fill%2Dvalue%20is%20the,be%20implied%20in%20some%20way.
* https://zarr.readthedocs.io/en/stable/spec/v2.html